sszgen/main.go

```diff
@@ -22,6 +23,11 @@ import (
 
 const bytesPerLengthOffset = 4
 
+type generationTarget struct {
+	name string
+	opts []string
+}
```

Comment:
This needs a comment, but what it basically does is allows options in the form of `StructName[option1,option2]`. Right now the unly supported option is `no-htr`.

Reply:
I'm not such a big fan of this style of passing arguments to the program, defining a sort of "micro-format" within flag arguments. Since it's a matter of taste, I'm curious what other folks think, but my instinct is that it would be a lot more clear, given that the only use case for this currently is disabling HTR generation, to set up a flag like `--no-htr=TypeA,TypeB`.